tests/onnx/test_low_opset_model.py

```diff
@@ -0,0 +1,125 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+import os
+
+import torch
+from torchvision import models
+import onnx
+
+from nncf.experimental.onnx.datasets.imagenet_dataset import infer_input_shape
+from tests.common.helpers import TEST_ROOT
+from tests.onnx.quantization.common import ModelToTest
+
+
+MODEL_NAMES = [
+    'resnet18',
+    'mobilenet_v2',
+    'mobilenet_v3_small',
+    'inception_v3',
+    'googlenet',
+    'vgg16',
+    'shufflenet_v2_x1_0',
+    'squeezenet1_0',
+    'densenet121',
+    'mnasnet0_5',
+]
+
+MODELS = [
+    models.resnet18(),
```

Comment:
All of these models have the same input shape and the same input_keys. It doen't make any sense to test all of them. Please use only two of them.
I wanted to see also models having different input_keys and different input_shape. Please, take a look at tests/onnx/data/models. Could you use [Mask R-CNN](https://github.com/onnx/models/tree/main/vision/object_detection_segmentation/mask-rcnn) and [yolov3](https://github.com/onnx/models/tree/main/vision/object_detection_segmentation/yolov3)